src/iqm/iqm_client/models.py

```diff
@@ -553,6 +627,16 @@ class RunRequest(BaseModel):
     heralding_mode: HeraldingMode = Field(HeraldingMode.NONE)
     """which heralding mode to use during the execution of circuits in this request."""
 
+    move_validation_mode: MoveGateValidationMode = Field(MoveGateValidationMode.STRICT)
+    """Which method of MOVE gate validation to use for circuit compilation. By default, the MOVE gate
+    validation is "strict": MOVE gates have to come in pairs without any prx gates between MOVE gates.
+    If set to "allow_prx", prx gates between MOVE gates are not validated. If set to "none", MOVE gate validation
+    is skipped altogether."""
```

Comment:
```suggestion
    """Which method of MOVE gate validation to use for circuit compilation."""
```
Better to not repeat the docstrings, the user can click on MoveGateValidationMode to see what the enum values mean.